source/includes/_tipser-elements.md

```diff
@@ -105,13 +104,15 @@ Insert below HTML on your page in the place where you want the `Store` element t
 <aside class="notify"><code>Store</code> Element is best inserted as a top level Element on a separate page and should contain the full content area for the best shopping experience.</aside>
 
 ## `Product` Element
-In order to insert a `Product` Element in your content, insert below code in your content.
+In order to insert a `Product` Element in your content, insert this code snippet in your content:
 
 ```html
 <div data-tipser-pid="5ba2334a781baa0001ccdf61" />
 ```
 
-Elements with attribute `data-tipser-pid` will be replaced with `Product` Element. Product ID is taken from the value of the attribute. By default a full inline product component is displayed (with product details, unique selling points and variant selection)
+Elements with attribute `data-tipser-pid` will be replaced with `Product` Element. Product ID is taken from the value of the attribute. The code snippet for a certain product can be generated with [tipser tools](#products-collections) automatically. 
```

Comment:
is `#produts-collections` a correct href to tipser-tools section? Shouldn't it be `#tipser-tools`?